flag_engine/engine.py

```diff
@@ -38,7 +42,7 @@ def get_environment_feature_state(environment: EnvironmentModel, feature_name: s
 def get_identity_feature_states(
     environment: EnvironmentModel,
     identity: IdentityModel,
-    override_traits: typing.List[TraitModel] = None,
+    override_traits: typing.Optional[typing.List[TraitModel]] = None,
```

Comment:
Is there a reason we use `typing.List` instead of just `list` here? I thought the typing capitalization is no longer necessary.

Reply:
Great question! Our SDK [still supports 3.8](https://github.com/Flagsmith/flagsmith-python-client/blob/main/pyproject.toml#L15) so, mainly, for that.